### ISSUE TITLE
Begin writing preliminary code for the AI

### ISSUE THREAD

**Issue by samduwu** [samduwu/nonogram-ai#7]
* Create a Jupyter Notebook in Google Colab to hold preliminary code (this will be pushed to this repo)
* Determine how to create said model in PyTorch and determine the best way to calculate loss
* If code is written for the above, figure out how to intake images as training data (determine how the images should be labeled, if necessary)
* Push the notebook to the repo
